build/workspace.bzl

```diff
@@ -13,3 +13,5 @@
 # limitations under the License.
 
 CRI_TOOLS_VERSION = "1.12.0"
+
+CNI_VERSION = "0.6.0"
```

Comment:
I'm ok with this, but being honest... I don't think this should live in bazel.

Reply:
I want to decouple bazel from the artifact generation.

Reply:
This change _just_ minimizes it down to ~ one place in bazel, which should be strictly better than the n places we had before. It definitely doesn't solve anything else.

It will be easier to derive it from another source later now that it is not hardcoded in many places. 
We could have a rule read it in from some other format. Or we can generate deb / rpm with another tool / set of rules or ....

Will leave that up to future PRs. Direction there seems pretty TBD and I'm low on bandwidth atm.

Reply:
Agree we're in a better place now rather than before. We could experiment with one central place for ~all well-known deps